jscomp/ml/printlambda.ml

```diff
@@ -177,6 +177,26 @@ let primitive ppf = function
   | Pfloatcomp(Cle) -> fprintf ppf "<=."
   | Pfloatcomp(Cgt) -> fprintf ppf ">."
   | Pfloatcomp(Cge) -> fprintf ppf ">=."
+  | Pnegbigint -> fprintf ppf "~"
+  | Paddbigint -> fprintf ppf "+"
+  | Psubbigint -> fprintf ppf "-"
+  | Pmulbigint -> fprintf ppf "*"
+  | Ppowbigint -> fprintf ppf "**"
+  | Pandbigint -> fprintf ppf "and"
+  | Porbigint -> fprintf ppf "or"
+  | Pxorbigint -> fprintf ppf "xor"
+  | Plslbigint -> fprintf ppf "lsl"
+  | Pasrbigint -> fprintf ppf "asr"
+  | Pdivbigint Safe -> fprintf ppf "/n"
+  | Pdivbigint Unsafe -> fprintf ppf "/nu"
```

Comment:
are these safe and unsafe operations supported in the current PR?

Also, what about `/n` and `/nu`: does this require binary operators, or how else are these triggered?

Reply:
I'm working on it.